pkg/packagekit/callTracer.py

```diff
@@ -0,0 +1,19 @@
+import json
+try:
+    from tracer.query import Query
+except ImportError:
+    raise ImportError("tracer not found")
```

Comment:
Please drop that -- converting a specific ImportError into a less generic one doesn't seem particularly useful?